src/ess/isissans/mantidio.py

```diff
@@ -32,10 +32,10 @@ def __getattr__(self, name: str) -> NoReturn:
     # Needed for type annotations
     MatrixWorkspace = object
 
-Period = NewType('Period', int)
+Period = NewType('Period', int | None)
```

Comment:
I was not sure if the `Optional` should be absorbed into the aliases like this. As it is now it is less explicit on the user side, but requires less changes. If we don't like it we can change later on?

Reply:
I prefer it like this, never really liked the `Optional` thing anyway